editor/Makefile

```diff
@@ -19,7 +19,7 @@ endif
 # https://github.com/google/closure-library/releases/
 closure = closure-library-20200517
 
-all : prep py js
+all : prep py js ketcher/dist/.touch
```

Comment:
The editor already has third-party dependencies: protobufs, closure, and the ORD proto classes. These are not set up with "make". Instructions are supplied in the README instead.

I think that way makes sense for such complex initializations where file modification times are not the best way to ensure consistency.

Reply:
While this makes sense to me, I just have one concern: if we make changes that require the newest version of the Ketcher repo, what's the best way to ensure that users pull and use this newest version? Should we add some versioning/version checking? 

(At the moment, the makefile pulls every time it's run, and only rebuilds when necessary.)

Reply:
If a dependency is something we control, then we can have an automated test that will block its changes if they break something downstream (which is common). We have no way to do that for third parties though, so usually it's better to have a manual step in the process.

Reply:
Ah I see. We're using our own fork of Ketcher, so maybe we do have this control. Automated tests would take a while though (maybe for later?); I can make this a manual step for now and put instructions in the readme.

Reply:
TODO (from mtng): tag our own version of Ketcher; in make, check that we're using this tag, and fail fast if we're not.

Reply:
^ in progress, reopening

Reply:
Finished -- added this check, added another check requiring Ketcher to be fully built, and added the installation process to GitHub CI.

@antonkast-google @skearnes would you guys be able to double-check the Makefile and CI? Will rebase and merge once everything's good!

Reply:
LGTM for the updates to `run_tests.yml`. It would be nice to have a one-shot script to install these dependencies at some point, but that's not blocking for this PR.